gcloud-java-storage/src/main/java/com/google/gcloud/storage/Blob.java

```diff
@@ -156,16 +156,18 @@ public Blob(Storage storage, BlobInfo info) {
 
   /**
    * Creates a {@code Blob} object for the provided bucket and blob names. Performs an RPC call to
-   * get the latest blob information.
+   * get the latest blob information. Returns {@code null} if the blob does not exist.
```

Comment:
Nit: we should probably standardize this javadoc and the javadoc for load below (either include the "Returns {@code null}..." sentence here and in the second `load` function, or just leave it in the `@return` statement like in the second `load` function.